lib/Doctrine/Common/DataFixtures/AbstractFixture.php

```diff
@@ -71,13 +71,25 @@ public function addReference($name, $object)
      *
      * @see Doctrine\Common\DataFixtures\ReferenceRepository::getReference
      *
-     * @param string $name
+     * @param string      $name
+     * @param string|null $class
      *
      * @return object
+     *
+     * @template T of object
+     * @psalm-param class-string<T>|null $class
+     * @psalm-return $class is null ? object : T
      */
-    public function getReference($name)
+    public function getReference($name, $class = null) // NEXT_MAJOR: Make $class mandatory
```

Comment:
We don't use such comment her ;)

```suggestion
    public function getReference($name, $class = null)
```

Reply:
No need for next major comment then ?
How do you remember what to do for 2.0 ?

Reply:
We look at the deprecation (it should say the parameter will become mandatory)